public/catch-js-errors.js

```diff
@@ -90,7 +90,7 @@
 
 	if ( isLocalStorageNameSupported() ) {
 		// Randomly assign 1% of users to log errors
-		if ( localStorage.getItem( 'log-errors' ) === undefined ) {
+		if ( ! localStorage.getItem( 'log-errors' ) ) {
```

Comment:
Could we add a follow up task to use `lib/localforage` instead of localStorage directly? We can avoid testing for localstorage then. @artpi Or perhaps this would be a good candidate for a user preference?